test/index.js

```diff
@@ -1206,6 +1206,37 @@ test('elem', function() {
   eq(Z.elem(0, Nothing), false);
 });
 
+test('foldMap', function() {
+  eq(Z.foldMap.length, 3);
+  eq(Z.foldMap.name, 'foldMap');
+
+  // Monoid instance for functions of type a -> a corresponding
+  // to reverse function composition
+  function DualEndo(f) {
+    if (!(this instanceof DualEndo)) return new DualEndo(f);
+    this.runEndo = f;
+  }
+  DualEndo['fantasy-land/empty'] = function() { return DualEndo(x => x); };
+  DualEndo.prototype['fantasy-land/concat'] = function(g) { return DualEndo(a => g.runEndo(this.runEndo(a))); };
```

Comment:
I like to name the parameter `other` to indicate that the argument is expected to be of the same type.